Cannot use useLocales or getClientLocales

### Describe the bug

The documentation lists these imports:

```ts
import { getClientLocales  } from 'remix-utils/get-client-locales'
import { useLocales } from 'remix-utils/use-locales'
```

However, these imports cannot be resolved. When I look at the `package.json` I see two different entries here: https://github.com/sergiodxa/remix-utils/blob/73b9bbbf05d94d53eb13edc995db6ee57576bf8d/package.json#L38

So I tried to this import: 

```ts
import { getClientLocales } from 'remix-utils/locales/server'
``` 

But that also does not seem to work (TS complains that it cannot find this export). This stuff used to work in `7.0.0-pre3`.

### Your Example Website or App

N/A

### Steps to Reproduce the Bug or Issue

See description above.

### Expected behavior

The export should either match the docs or the docs should match the `package.json` and it should work :)

### Screenshots or Videos

_No response_

### Platform

- OS: macOS
- Browser: N/A
- Version: 7.0.1


### Additional context

_No response_

I've got same with `import { eventStream } from 'remix-utils/sse/server'`:

```
Cannot find module 'remix-utils/sse/server' or its corresponding type declarations.
```

--- a/README.md
+++ b/README.md
@@ -836,8 +836,8 @@ This hooks lets you get the locales returned by the root loader. It follows a si
 You can combine it with `getClientLocal` to get the locales on the root loader and return that. The return value of `useLocales` is a `Locales` type which is `string | string[] | undefined`.
 
 ```ts
-import { useLocales } from "remix-utils/use-locales";
-import { getClientLocales } from "remix-utils/get-client-locales";
+import { useLocales } from "remix-utils/locales/react";
+import { getClientLocales } from "remix-utils/locales/server";
 
 // in the root loader
 export async function loader({ request }: LoaderArgs) {
@@ -960,7 +960,7 @@ When a header is found that contains a valid IP address, it will return without 
 This function let you get the locales of the client (the user) who originated the request.
 
 ```ts
-import { getClientLocales } from "remix-utils/get-client-locales";
+import { getClientLocales } from "remix-utils/locales/server";
 
 export async function loader({ request }: LoaderArgs) {
 	// using the request
@@ -975,7 +975,7 @@ The return value is a Locales type, which is `string | string[] | undefined`.
 The returned locales can be directly used on the Intl API when formatting dates, numbers, etc.
 
 ```ts
-import { getClientLocales } from "remix-utils/get-client-locales";
+import { getClientLocales } from "remix-utils/locales/server";
 export async function loader({ request }: LoaderArgs) {
 	let locales = getClientLocales(request);
 	let nowDate = new Date();

